index.html

```diff
@@ -349,9 +354,9 @@ <h2>Project related news</h2>
           <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
             <div class="news-wrapper">
               <img src="img/website.png" alt="news icon" class="news-icon" />
-              <p class="news-source">Law Commission</p>
-              <p class="news-article"><a href="http://www.lawcom.govt.nz/news/relationship-property-review-underway">Relationship property review underway</a></p>
-              <p class="news-date">27 November 2017</p>
+              <p class="news-source">NZ Herald</p>
+              <p class="news-article"><a href="http://www.nzherald.co.nz/business/news/article.cfm?c_id=3&objectid=10870095">Diana Clement: Splitting property not always straightforward.</a></p>
+              <p class="news-date">9 March 2013</p>

```

Comment:
Put the articles in order of publication date

Reply:
Good idea - done.